bin/oio-blob-mover

```diff
@@ -16,12 +16,34 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+import argparse
+
 from oio.blob.mover import BlobMover
 from oio.common.daemon import run_daemon
-from oio.common.configuration import parse_options
-from optparse import OptionParser
+
+def make_arg_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('config', help="Configuration file")
+    parser.add_argument('--number', type=int,
+                        help="Number of chunk to move")
+    parser.add_argument('--usage-target', type=int,
+                        help="Percentage of disk usage target")
+    parser.add_argument('--daemon', type=int,
+                        help="Interval to run the mover in seconds.")
```

Comment:
This is ambiguous. The option is named "daemon" but we ask for a duration.
I think it would be clearer to make two arguments:
- a boolean to make the process a daemon;
- an integer to change the duration between passes (with a good default value).